autoprotocol/container_type.py

```diff
@@ -1170,6 +1170,42 @@ def row_count(self):
     cat_no="4513",
 )
 
+#:
+ROUND384LVNC = ContainerType(
+    name="384-well round-bottom low-volume Corning-4512 plate",
+    well_count=384,
+    well_depth_mm=6.58,
+    well_volume_ul=Unit(35.0, "microliter"),
```

Comment:
@cj-bonilla, I asked @polarpine and you need
```
true_max_vol_ul: 35
well_volume_ul: 20
```

Reply:
Perfect, thanks will address this!